frontend/src/components/explore-page/explore-page.component.js

```diff
@@ -7,8 +7,35 @@ import Loader from '../loader/loader.component';
 import GoogleMap from '../google-map/google-map.component';
 import { CircleComponent } from '../util-components/util-components.component';
 
+const CircleArray = ({ array, selectedFilter, setSelectedFilter}) => {
+  return array.map((circleComp) => {
+    return (
+      <CircleComponent
+        title={circleComp.title}
+        coordinatedByText= {circleComp.coordinatedByText}
+        selected={selectedFilter === circleComp.id}
+        onClick={() => setSelectedFilter(circleComp.id)}
+      />
+    )
+  })
+}
+
 const ExplorePage = () => {
   const dispatch = useDispatch();
+  const [selectedFilter, setSelectedFilter] = useState(-1);
```

Comment:
Won't this filter all Circle by default?

Reply:
This is for the when clicking a circle, by default i want no circle to be selected, so the value of selectedFilter is `-1`, when clicking a circle, we trigger setSelectedFilter which in tern turns selectedFilter to the id and highlights it